### PR TITLE
Update service-premium-scale-out.md

### PR DESCRIPTION
**Pull request by gnair-ms**
Add a note for Analyze In Excel scenario.

### CODE REVIEW

File: powerbi-docs/enterprise/service-premium-scale-out.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ By default, scale-out is enabled for your tenant, but it's not enabled for seman
     | App, library or service  | Version |
     |--------------------------|---------|
     | Microsoft Analysis Services OLE DB Provider for Microsoft SQL Server (MSOLAP) | 16.0.20.201 (March 2022) |
+    | Analysze in Excel for Power BI Semantic models (MSOLAP) | 16.0.122.27 |
```
Comment (vailavar) [inline powerbi-docs/enterprise/service-premium-scale-out.md:71]: typo: Analyze

File: powerbi-docs/enterprise/service-premium-scale-out.md [hunk 2]
```diff
@@ -145,6 +146,8 @@ Power BI semantic model scale-out is enabled by default for a tenant. Power BI t
     * Restoring a semantic model using the public XMLA endpoint.
 
 * Disabling [Large semantic model storage format](service-premium-large-models.md) disables scale-out and loses all sync information.
+  
+* When multiple read-only replicas are active, Analyze in Excel for Semantic Models feature using MSOLAP versions earlier than 16.0.122.27 can result in frequest connectivity issues with error message "session ID cannot be found". This issue can be resolved by using direct Power BI Workspace connections (powerbi://) or by upgrading to latest MSOLAP provider.
```
Comment (vailavar) [inline powerbi-docs/enterprise/service-premium-scale-out.md:150]: typo: frequent

**Review by Kay-Personal — CHANGES_REQUESTED** [review]
Please use the following wording: 

When multiple read-only replicas are active, using Analyze in Excel for Semantic Models with MSOLAP versions earlier than 16.0.122.27 can cause frequent connectivity issues with the error message "session ID cannot be found". To fix this issue, either use a direct Power BI workspace connection (powerbi://) or upgrade to the latest MSOLAP provider.

File: powerbi-docs/enterprise/service-premium-scale-out.md [hunk 3]
```diff
@@ -68,6 +68,7 @@ By default, scale-out is enabled for your tenant, but it's not enabled for seman
     | App, library or service  | Version |
     |--------------------------|---------|
     | Microsoft Analysis Services OLE DB Provider for Microsoft SQL Server (MSOLAP) | 16.0.20.201 (March 2022) |
+    | Analyze in Excel for Power BI Semantic models (MSOLAP) | 16.0.122.27 |
```
Comment (vailavar) [inline powerbi-docs/enterprise/service-premium-scale-out.md:71]: nit: Models for case-consistency?

Comment (rahulten) [reply powerbi-docs/enterprise/service-premium-scale-out.md:71]: Power BI docs use the term, "Power BI semantic models" (small s, small m)

**Comment by gnair-ms** [comment]
> Please use the following wording:
> 
> When multiple read-only replicas are active, using Analyze in Excel for Semantic Models with MSOLAP versions earlier than 16.0.122.27 can cause frequent connectivity issues with the error message "session ID cannot be found". To fix this issue, either use a direct Power BI workspace connection (powerbi://) or upgrade to the latest MSOLAP provider.

**Comment by gnair-ms** [comment]
> > Please use the following wording:
> > When multiple read-only replicas are active, using Analyze in Excel for Semantic Models with MSOLAP versions earlier than 16.0.122.27 can cause frequent connectivity issues with the error message "session ID cannot be found". To fix this issue, either use a direct Power BI workspace connection (powerbi://) or upgrade to the latest MSOLAP provider.

**Comment by KesemSharabi** [comment]
Hi @***@***.***> (+ @***@***.***> for awareness),

We decided not to put Excel versions in the document because it'll be difficult to maintain. In light of this I'm closing the PR.

Also, in future please make documentation contributions in the private repo (powerbi-docs-pr<https://github.com/MicrosoftDocs/powerbi-docs-pr>). There's more info in Contribute to Microsoft Learn.loop<https://microsoft.sharepoint.com/:fl:/s/4501ccc2-16b2-4f90-8aa1-94a385c56166/EeLnb_7eRxVDkcm8VJ3duIIBhd0m52XVLBTMe8jJixydNQ?e=IKctOp&nav=cz0lMkZzaXRlcyUyRjQ1MDFjY2MyLTE2YjItNGY5MC04YWExLTk0YTM4NWM1NjE2NiZkPWIhb3lDOTQ3ejlnay0xTFU1RVF3RG5tWG1VUHNSYjN3cE1oZVFyYWNaMzJaY3FpX014eUhBRVRaSF9wZzdMWU5tUyZmPTAxWkg0VUNOSEM0NVg3NVhTSENWQlpEU040S1NPNTNPRUMmYz0lMkYmZmx1aWQ9MSZhPUxvb3BBcHAmeD0lN0IlMjJ3JTIyJTNBJTIyVDBSVFVIeHRhV055YjNOdlpuUXVjMmhoY21Wd2IybHVkQzVqYjIxOFlpRnZlVU01TkRkNk9XZHJMVEZNVlRWRlVYZEVibTFZYlZWUWMxSmlNM2R3VFdobFVYSmhZMW96TWxwamNXbGZUWGg1U0VGRlZGcElYM0JuTjB4WlRtMVRmREF4V2tnMFZVTk9Rek5KVWt4RFVrZFpTalZXUVV0RlQwSTFNek5MTTBoR1NsayUzRCUyMiUyQyUyMmklMjIlM0ElMjIwYzY0ZWViNy00OTYxLTQzODMtOTFhMC05YjNmM2FlZWZiODUlMjIlN0Q%3D>.

Thanks,

Kesem

________________________________
From: gnair-ms ***@***.***>
Sent: Tuesday, December 10, 2024 02:36
To: MicrosoftDocs/powerbi-docs ***@***.***>
Cc: Kesem Sharabi ***@***.***>; Assign ***@***.***>
Subject: [MicrosoftDocs/powerbi-docs] Update service-premium-scale-out.md (PR #5166)


Add a note for Analyze In Excel scenario.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/MicrosoftDocs/powerbi-docs/pull/5166

Commit Summary

  *   5926626<https://github.com/MicrosoftDocs/powerbi-docs/pull/5166/commits/5926626ee964e8946e1cbfb1a5cb12b353671871> Update service-premium-scale-out.md

File Changes

(1 file<https://github.com/MicrosoftDocs/powerbi-docs/pull/5166/files>)

  *   M powerbi-docs/enterprise/service-premium-scale-out.md<https://github.com/MicrosoftDocs/powerbi-docs/pull/5166/files#diff-c33017d52afc7d20ca182bc0f831a26cb4f95fdb7d26edc7c584d98c32117c5d> (3)

Patch Links:

  *   https://github.com/MicrosoftDocs/powerbi-docs/pull/5166.patch
  *   https://github.com/MicrosoftDocs/powerbi-docs/pull/5166.diff

**Comment by Kay-Personal** [comment]
Kesem, MSOLAP 16.0.122.27 is not an Excel version but a version of our client libs, and it is a constant that doesn't need to be maintained any further because it is the first version when the fix was made available. Maybe 10 years from now we can remove this version number, but if the version check is irrelevant at that time, the entire note would be irrelevant and could be removed altogether. For now, the version number is important because it enables customers to check that they have a required version that solves the problem. Not sure how we would accomplish this without stating the version number.

**Pull request closed without merging**